1/ToDoApp/src/backend/schema/typeDefs.js

```diff
@@ -0,0 +1,32 @@
+const { gql } = require('apollo-server');
+// A schema is a collection of type definitions (hence "typeDefs")
+// that together define the "shape" of queries that are executed against
+// your data.
+const typeDefs = gql`
+  type Todo {
+    id: Int!
+    message: String
+    finished: Boolean
+    creator: [User]
```

Comment:
We can have many creator for a todo? :thinking:

Either this
```suggestion
    creators: [User]
```
or this:
```suggestion
    creator: User
```
I would say..